spec/models/framework/miso_fields_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe Framework::MisoFields do
+  let(:framework_short_name) { 'RM3787' }
+
+  let(:miso_fields) { Framework::MisoFields.new(framework_short_name) }
+
+  describe '#filename' do
```

Comment:
does this need publicly exposing and tested?

Reply:
No. I can't remember why it was now. I'll remove.